### PR TITLE
add mode to intview

### PR DESCRIPTION
**Pull request by yij793**
address: https://nanopay.atlassian.net/browse/CPF-3507
note: the intview class visibility RO isn't working. please check the change I made.
@TW80000

### CODE REVIEW

File: src/foam/u2/IntView.js [hunk 1]
```diff
@@ -44,6 +44,7 @@ foam.CLASS({
       this.SUPER(p);
       this.min = p.min;
       this.max = p.max;
+      this.mode = p.mode;
```
Comment (TW80000) [inline src/foam/u2/IntView.js:47]: This doesn't make sense to me. `p` is a `Property` and afaik, `Property`-ies don't have a `mode` property.

Comment (yij793) [reply src/foam/u2/IntView.js:47]: > This doesn't make sense to me. `p` is a `Property` and afaik, `Property`-ies don't have a `mode` property.

We have `mode` property at `View` class in Element.js line: 2699
`    class: 'Enum',
      of: 'foam.u2.DisplayMode',
      name: 'mode',`
and the intView is extend from View

Comment (TW80000) [reply src/foam/u2/IntView.js:47]: Right, so `this.mode` makes sense, but it's `p.mode` that doesn't make sense to me.

**Review by KernelDeimos — COMMENTED** [review]
There is another IntView at `foam.u2.view.IntView` rather than `foam.u2.IntView`. The one in the `view` package extends `foam.u2.view.ModeAltView` which has this value for documentation:
> A view that shows one of several views, depending on the display mode.

Does foam.u2.view.IntView have the same behaviour as this change, or is this something different?

**Pull request closed without merging**